tests/end_to_end/global_decls/irix-o2-allglobals-out.c

```diff
@@ -7,22 +7,30 @@ MIPS2C_UNK static_fn(struct A *);                   /* static */
 extern f32 extern_float;
 struct A static_A = {
     1,
-    {1, 2, 3, 4, 5},
+    { 1, 2, 3, 4, 5 },
     {
-        {{1.0f, 2.0f, 3.0f, 4.0f}, {5.0f, 6.0f, 7.0f, 8.0f}, {9.0f, 0.0f, 1.0f, 2.0f}},
-        {{1.5f, 2.5f, 3.5f, 4.5f}, {5.5f, 6.5f, 7.5f, 8.5f}, {9.5f, 0.5f, 1.5f, 2.5f}},
+        {
+            { 1.0f, 2.0f, 3.0f, 4.0f },
+            { 5.0f, 6.0f, 7.0f, 8.0f },
+            { 9.0f, 0.0f, 1.0f, 2.0f },
+        },
+        {
+            { 1.5f, 2.5f, 3.5f, 4.5f },
+            { 5.5f, 6.5f, 7.5f, 8.5f },
+            { 9.5f, 0.5f, 1.5f, 2.5f },
+        },
     },
-    {NULL, &static_int},
+    { NULL, &static_int },
     &static_bss_A,
 };
 struct A *static_A_ptr = &static_A;
-s32 static_array[3] = {2, 4, 6};
+s32 static_array[3] = { 2, 4, 6 };
 struct A static_bss_A;
 s32 static_bss_array[3];
 s32 static_int;
-s32 unused_static_array[3] = {8, 0xA, 0xC};
-s32 static_ro_array[3] = {7, 8, 9};                 /* const */
-s32 unused_static_ro_array[3] = {0xA, 0xB, 0xC};    /* const */
+s32 unused_static_array[3] = { 8, 0xA, 0xC };
+s32 static_ro_array[3] = { 7, 8, 9 };               /* const */
+s32 unused_static_ro_array[3] = { 0xA, 0xB, 0xC };  /* const */
```

Comment:
would be nice to change the test to have one of these arrays be s16, one s32, one u32, with one negative member in each. (involves changing the orig.c file and also the .s files manually)

Reply:
Could you check if I did this correctly?

Reply:
Looks good to me.